azext_edge/tests/edge/support/create_bundle_int/helpers.py

```diff
@@ -349,3 +410,10 @@ def run_bundle_command(
             walk_result[built_path]["files"].append(file_name)
 
     return walk_result
+
+
+def get_bundle_path(
+    tracked_files: List[str],
+) -> str:
+    # find bundle path from tracked_files that with .zip extension
+    return next((file for file in tracked_files if file.endswith(".zip")), None)
```

Comment:
? are you trying to keep the bundle arg?

Reply:
okie, I was trying to reduce the output from run command